readme.txt

```diff
@@ -1,37 +1,35 @@
 Tested up to: 5.7
 Requires PHP: 5.6
 Stable tag: trunk
 License: GPLv2 or later
 License URI: http://www.gnu.org/licenses/gpl-2.0.html
 
-Enable CDN URLs for your static assets such as images, CSS or JavaScript files.
 
 == Description ==
 
-[ImageEngine’s Image CDN](https://imageengine.io/?utm_source=wordpress.org&utm_medium=page&utm_term=wp-imageengine&utm_campaign=wp-imageengine) plugin accelerates your WordPress or WooCommerce performance by optimizing static images and other assets and delivering them through the ImageEngine content delivery network. The plugin automatically tailors images specifically to the end user's smartphone, tablet or desktop device model and browser, converts them to an optimal image format (e.g. WebP or JPEG 2000), and delivers them via a CDN. To use this plugin with the ImageEngine CDN, get a [free trial account here](https://imageengine.io/signup?utm_source=wordpress.org&utm_medium=page&utm_term=wp-imageengine&utm_campaign=wp-imageengine).
+[ImageEngine’s Image CDN](https://imageengine.io/?utm_source=wordpress.org&utm_medium=page&utm_term=wp-imageengine&utm_campaign=wp-imageengine) plugin accelerates your WordPress or WooCommerce performance by optimizing images, converting them to WebP, JPEG2000, or AVIF, and delivering them through the ImageEngine content delivery network. The result is smaller image payload, faster page loading, improved Google PageSpeed Insights scores (Speed Index, Largest Contentful Paint, Time to Interactive), and a better user experience leading to more conversions or sales. To use this plugin with the ImageEngine Optimizer CDN, get a [free trial account here](https://imageengine.io/signup?utm_source=wordpress.org&utm_medium=page&utm_term=wp-imageengine&utm_campaign=wp-imageengine).
 
-= How ImageEngine Works =
+https://vimeo.com/496085924
 
-The plugin works by rewriting the URLs to your assets (images, javascript, css, etc), switching your domain for the CDN's domain. Unlike other CDN plugins, the Image CDN plugin makes it simple to test your configuration before enabling it.
+= How ImageEngine Works =
 
-* When a visitor requests a page, ImageEngine dynamically pulls images from your WordPress origin and optimizes them.
-* At the CDN edge, ImageEngine uses device detection to identify the requesting devices and browser characteristics.
-* Based on this information, ImageEngine will resize, compress and encode images so they are best suited to the end users' device.
-* The optimized image will then be delivered at the ImageEngine CDN edge. Subsequent requests are served instantly with cached assets stored on ImageEngine’s global CDN.
+This plugin rewrites your image URLs to include the ImageEngine Delivery Address you recieve when you sign up for an ImageEngine account. This rewrite will allow ImageEngine to access your original images, instantly optimize and convert them, and deliver via the ImageEngine CDN. After configuring and enabling the plug in, image are delivered this way:
```

Comment:
Not sure if its worth mentioning, but it also optimizes non-image content, serving CSS, Javascript and Fonts with Brotli compression.

Reply:
added a paragraph